src/WishListPage/wishListPage.html

```diff
@@ -0,0 +1,179 @@
+                            <h1>₹2000</h1>
+                        </div>   
+                    
+                    </div> 
+                    <div class="card-footer">
+                        <div class="card-footer-view">
+                            <button>Go to Cart</button>
+                        </div>
+                    </div>
+                    <span class="material-icons-round badge topright-badge badge-warning">
+                        favorite_border
+                        </span>
+                </div>
+                <div class="card cart-card">
+                    <img class="card-img" src="../../asset/img/ervan-m-wirawan-6r280_Z7Efc-unsplash.jpg" alt="Food Image" srcset=""/>
+                
+                    <div class="card-content">
+                        <div class="card-body">
+                            Men Premium Shoe
+                        </div>
+                        <div class="card-title">
+                            <h1>₹2000</h1>
+                        </div>   
+                    
+                    </div> 
+                    <div class="card-footer">
+                        <div class="card-footer-view">
+                            <button>Go to Cart</button>
+                        </div>
+                    </div>
+                    <span class="material-icons-round badge topright-badge badge-warning">
+                        favorite_border
+                        </span>
+                </div>
+                <div class="card cart-card">
+                    <img class="card-img" src="../../asset/img/logan-weaver-PdO0sdaLcdw-unsplash.jpg" alt="Food Image" srcset=""/>
+                
+                    <div class="card-content">
+                        <div class="card-body">
+                            Men Premium Shoe
+                        </div>
+                        <div class="card-title">
+                            <h1>₹2000</h1>
+                        </div>   
+                    
+                    </div> 
+                    <div class="card-footer">
+                        <div class="card-footer-view">
+                            <button>Go to Cart</button>
+                        </div>
+                    </div>
+                    <span class="material-icons-round badge topright-badge badge-warning">
+                        favorite_border
+                        </span>
+                </div>
+                <div class="card cart-card">
+                    <img class="card-img" src="../../asset/img/paul-gaudriault-a-QH9MAAVNI-unsplash.jpg" alt="Food Image" srcset=""/>
+                
+                    <div class="card-content">
+                        <div class="card-body">
+                            Men Premium Shoe
+                        </div>
+                        <div class="card-title">
+                            <h1>₹2000</h1>
+                        </div>   
```

Comment:
dont use jpeg , if the size is larger , it will hinder the performance..!!

Reply:
okay will take care of the image type